simple-pod.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
You probably didn't mean to commit this file.

Reply:
right i did not

Reply:
updated, removed the file, thanks